style_guides/js_style_guide.md

```diff
@@ -441,142 +550,151 @@ Keep your functions short. A good function fits on a slide that the people in
 the last row of a big room can comfortably read. So don't count on them having
 perfect vision and limit yourself to ~15 lines of code per function.
 
-## Return early from functions
+## Use "rest" syntax rather than built-in `arguments`
 
-To avoid deep nesting of if-statements, always return a function's value as early
-as possible.
-
-*Right:*
+For expressiveness sake, and so you can be mix dynamic and explicit arguments.
```

Comment:
I think the example should demonstrate how dynamic and explicit arguments can be mixed, to illustrate this explanation.